presto-native-execution/Makefile

```diff
@@ -77,23 +77,23 @@ TypeSignature:		#: Build the Presto TypeSignature parser
 	cd presto_cpp/main/types; $(MAKE) TypeSignature
 
 format-fix: 			#: Fix formatting issues in the master branch
-	velox/scripts/check.py format master --fix
+	pushd "$$(pwd)/.." && presto-native-execution/velox/scripts/check.py format master --fix && popd
```

Comment:
@mshang816 Why do we need these change? Isn't this the same as `velox/scripts/check.py format master --fix`

Reply:
> @mshang816 Why do we need these changes? Isn't this the same as `velox/scripts/check.py format master --fix`

@majetideepak velox/scripts/check.py is designed to be run under a git root. without the trick in the PR, when you run something like "make format-fix" the script will have a working directory at presto-native-execution then you will see many errors. There is a work-around running the script from presto root with `presto-native-execution/velox/scripts/check.py format master --fix`
This PR just tries to change the current directory to presto root then run script in the way mentioned above. I'd admit it's not a beautiful solution but it's a quick fix for people to run format check using make before we have a better fix in the check.py. Note that check.py is also used by velox, for any fix we need to make sure it will work in velox too.

Reply:
Thanks for the clarification.